cloudant-client/src/main/java/com/cloudant/client/org/lightcouch/internal/CouchDbUtil.java

```diff
@@ -127,12 +127,15 @@ public static String removeExtension(String fileName) {
     }
 
     public static String streamToString(InputStream in) {
-        Scanner s = new Scanner(in, "UTF-8");
-        s.useDelimiter("\\A");
-        String str = s.hasNext() ? s.next() : null;
-        close(in);
-        s.close();// mdb
-        return str;
+        try {
+            Scanner s = new Scanner(in, "UTF-8");
+            s.useDelimiter("\\A");
+            String str = s.hasNext() ? s.next() : null;
+            s.close();// mdb
```

Comment:
Actually this might be outside the scope of this PR but I'm confused as to what this method is supposed to do. It appears to use a regex to match all of the text (`\A` matches "The beginning of the input") as a rather obscure way of reading a stream as a string. As long as we are happy to have `IOUtils` as a dependency, I would suggest we use the same one-liner as `HttpConnection.responseAsString`